Cache/IO.lean

```diff
@@ -70,7 +70,8 @@ def getPackageDirs : IO PackageDirs := return .ofList [
   ("Mathlib", if ← isMathlibRoot then "." else mathlibDepPath),
   ("Aesop", LAKEPACKAGESDIR / "aesop"),
   ("Std", LAKEPACKAGESDIR / "std"),
-  ("Qq", LAKEPACKAGESDIR / "Qq")
+  ("Qq", LAKEPACKAGESDIR / "Qq"),
+  ("Extras", ".")
```

Comment:
Should this be namespaced in some way, e.g. `MathlibExtras`? Will this cause conflicts with other packages with an `Extras` folder, or a package named `Extras`?

Reply:
Okay, replaced all `Extras` with `MathlibExtras`. I'll do another round of CI to make sure it is still putting things in findable places!